src/sensor/protocoldetector.cpp

```diff
@@ -147,8 +146,8 @@ bool ProtocolDetector::checkSerial(LinkConfiguration& linkConf)
 bool ProtocolDetector::checkUdp(LinkConfiguration& linkConf)
 {
     // To find a ping, we this message on a link, then wait for a reply
-    ping_msg_ping1D_empty req;
-    req.set_id(Ping1DNamespace::Firmware_version);
+    ping_message_general_request req;
```

Comment:
Are we ready to make this change?

Reply:
Needs 3.27 to be release first.

Reply:
@patrickelectric we should format the request according to the firmware version. We can remove the retro-support after ~6 months.

Reply:
`general_request` is ok to use in `ProtocolDetector`, the only problem that is know after Ping1D release is while using `general_request` with `profile`. This patch is already compatible with old firmware versions.